lib/charms/operator_libs_linux/v0/systemd.py

```diff
@@ -174,3 +175,12 @@ def service_resume(service_name: str) -> bool:
     _service("unmask", service_name)
     _service("enable", service_name, now=True)
     return service_running(service_name)
+
+
+def daemon_reload() -> bool:
+    """Reload systemd manager configuration."""
+    proc = subprocess.Popen(["systemctl", "daemon-reload"], **_popen_kwargs())
+    for line in iter(proc.stdout.readline, ""):
+        logger.debug(line)
+    proc.wait()
+    return proc.returncode == 0
```

Comment:
Would it make more sense to just make "action" an optional argument for _service, and pass this through the _service command?

Since _service is an internal implementation detail, we could rename it something more generic. Perhaps `call`, or `ctl`.

Reply:
Agree, this was stupid. PR to follow...